markdown/2017-10-23.md

```diff
@@ -0,0 +1,179 @@
+- **Where:** Huxley 341/342
+- **Link:** [Facebook](https://www.facebook.com/events/2207747869451881/)
+
+G-Research is a leading quantitative research and technology company. We
+apply scientific techniques to find patterns in large, noisy and real-world
+data sets, using the latest statistical and "big data" analysis methodologies
+to predict global financial markets.
+ 
+Please join us for our annual coding challenge where you will compete in
+teams in a series of progressively more difficult sentiment analysis
+challenges.
+
+This is a great chance to test your problem solving and coding skills and a
+chance to win a &#163;50 Amazon vouchers per team member (plus prizes for
+runners up!).
+ 
+To take part all you need is at least one laptop per team and have a
+development environment set up for C#, Java or Python and you must be able to
+provide your own internet access.
+ 
+Afterwards we will provide food and drinks and a chance for you to talk to
+our employees informally about internships, graduate roles and life at
+G-Research!
+ 
+To register your interest please email <graduates@gresearch.co.uk>. You can
+register alone or as part of a team of up to 4 members, if you register
+individually you will be allocated a team at the event.
+
+---
+
+Our sponsors would like to let you know:
+
+# Microsoft Future Decoded
+
+Microsoft is running an event called Future Decoded.
+
+- **When:** 31st October-1st November 2017
+- **Where:** ExCeL London, E16 1XL, London
+- **Link:** [More information](https://futuredecoded.com)
+
+Back for a fourth year, Microsoft Future Decoded will focus on current and
+emerging trends at the cross over of business and technology. With cloud, big
+data and artificial intelligence already rewriting the rules of business and
+competition in every industry, it is essential that whatever your role, you
+understand the impact of technology on your business and on your career.
+
+With automation set to change the way we work in the coming years, Microsoft
+Future Decoded will provide insight into the skills and knowledge you will
+need in business and in technology to thrive in today's world of cloud, data,
+AI, Blockchain and virtual worlds. Plus, learn how to prepare for the seismic
+impact of tomorrow's computing, based on DNA and quantum computers.
+
+*Please note that this event is not organised by DoCSoc*
+
+# Microsoft Imagine
+
+Microsoft Imagine connects students with the tools, resources and experiences
+they need to elevate their skills for today's working world. Whether it's
+building a game, designing an app or launching a project, Microsoft Imagine
+helps students develop their ideas and boldly bring them to life.
+
+[Find out more about Microsoft Imagine](https://imagine.microsoft.com/en-us)
+
+---
+
+# AWS Workshoe
+
+Imperial College Advanced Hackspace (ICAH) are running an AWS workshop:
+
+- **When:** 13.00-17.00, 1st November 2017
+- **Where:** Royal School of Mines (RSM)
```

Comment:
Is there a room?

Reply:
no room given. Presumably they'll signpost